thirdeye/thirdeye-frontend/app/utils/utils.js

```diff
@@ -32,24 +32,54 @@ export function checkStatus(response, mode = 'get', recoverBlank = false) {
  * Formatter for the human-readable floating point numbers numbers
  */
 export function humanizeFloat(f) {
```

Comment:
Can you include an `@example` here for clarity. It's not immediately understandable from the inline comments below.